builtin/credential/radius/path_login.go

```diff
@@ -0,0 +1,157 @@
+package radius
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+	"time"
+
+	"layeh.com/radius"
+
+	"github.com/hashicorp/vault/helper/policyutil"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathLogin(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: `login/(?P<username>.+)`,
```

Comment:
Can we use `framework.OptionalParamRegex` here? That way, `username` can only be mentioned in the body instead of the URL, when desired.